c3po/db/models/link.py

```diff
@@ -16,10 +16,14 @@ class Link(Base):
 
     post_count = Column("post_count", Integer)
     likes_count = Column("likes_count", Integer)
+    views = Column("views", BigInteger)
```

Comment:
Using BigInteger for views so that Gangnam style doesn't break your system 😆

Reply:
Exactly :laughing: